.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,6 +26,7 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 - [ ] I have updated the [HISTORY.rst](https://github.com/simai-ml/MAPIE/blob/master/HISTORY.rst) and [AUTHORS.rst](https://github.com/simai-ml/MAPIE/blob/master/AUTHORS.rst) files
 - [ ] Linting passes successfully : `make lint`
```

Comment:
What's the difference between `make link` and `make black`? Are we sure that's the correct order?

Reply:
You're right, I'll put it on earlier.